test/jdk/java/awt/image/multiresolution/MultiDisplayTest.java

```diff
@@ -87,19 +130,17 @@ private static BufferedImage generateImage(int scale, Color c) {
         g.setColor(Color.WHITE);
         Font f = g.getFont();
         g.setFont(new Font(f.getName(), Font.BOLD, scale * 48));
-        g.drawChars((scale + "X").toCharArray(), 0, 2, scale * W / 2, scale * H / 2);
-
+        g.drawChars((scale + "X").toCharArray(), 0, 2,
+                scale * W / 2, scale * H / 2);
         return image;
     }
 
     private static class ParentFrame extends Frame {
-
         public ParentFrame() {
             EventQueue.invokeLater(this::CreateUI);
         }
```

Comment:
This looks weird…

As far as I can see, the frame is created when Start button is clicked, which implies this operation is running on EDT. There's no reason to use `invokeLater`, all the code from `CreateUI` method could be placed into the constructor.